run_iwyu_tests.py

```diff
@@ -69,7 +69,8 @@ def setUp(self):
                                     self.CheckAlsoExtension('-fnreturn.h'),
                                     self.CheckAlsoExtension('-typedefs.h'),
                                     self.CheckAlsoExtension('-d2.h')],
-      'keep_mapping.cc': [self.CheckAlsoExtension('-public.h'), 
+      'keep_includes.c': ['--keep=./*/*/keep_includes-d1.h'],
```

Comment:
Hm, I wonder if the Windows implementation of `GlobMatchesPath` can take this pattern...

Sorry to go back and forth on this, but maybe it's safer to use something `--keep=keep_includes-*.h`.